org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AsyncServerMarkerUpdaterJob.java

```diff
@@ -25,24 +25,21 @@
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.Status;
-import org.sonarlint.eclipse.core.internal.TriggerType;
 import org.sonarlint.eclipse.core.internal.resources.SonarLintProject;
 import org.sonarlint.eclipse.core.internal.tracking.Trackable;
 
-public class MarkerUpdaterJob extends AbstractSonarProjectJob {
+public class AsyncServerMarkerUpdaterJob extends AbstractSonarProjectJob {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This class has 7 parents which is greater than 5 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)